test/rewrites.lean

```diff
@@ -130,3 +132,9 @@ info: Try this: rw [@AddCommMonoidWithOne.add_comm]
 #guard_msgs in
 example (n : ℕ) : let y := 3; n + y = 3 + n := by
   rw?!
+
+-- Check that the same lemma isn't used multiple times.
+-- This used to report e.g. two redundant copies of `Nat.add_assoc`.
+lemma test (n m : ℕ) : n + m + 1 = m + n + 1 := by
```

Comment:
Can we add a guard msgs here or something to make this an actual test test? (Maybe abstracting away from nat so not too many results are returned

Reply:
Done!